<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrthographicSubWindow</class>
 <widget class="QWidget" name="OrthographicSubWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Orthographic View</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <item>
    <widget class="View::MapView2D" name="centralView" native="true">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Orientation:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbTop">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="text">
        <string>Top</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbFront">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="text">
        <string>Front</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbSide">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="text">
        <string>Side</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>View::MapView2D</class>
   <extends>QWidget</extends>
   <header location="global">mapview2d.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>rbTop</sender>
   <signal>clicked()</signal>
   <receiver>OrthographicSubWindow</receiver>
   <slot>setTop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbFront</sender>
   <signal>clicked()</signal>
   <receiver>OrthographicSubWindow</receiver>
   <slot>setFront()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbSide</sender>
   <signal>clicked()</signal>
   <receiver>OrthographicSubWindow</receiver>
   <slot>setSide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setTop()</slot>
  <slot>setFront()</slot>
  <slot>setSide()</slot>
 </slots>
</ui>
